Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM nvidia/cuda:11.3.1-cudnn8-devel-ubuntu20.04
+MAINTAINER Peter Willemsen <peter@codebuffet.co>
+RUN echo "Installing dependencies..." && \
+    apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y curl wget sudo git build-essential cmake pkg-config liblzma-dev libbz2-dev zlib1g-dev libssl-dev zsh clang && \
+    apt-get dist-upgrade -y && \
+	rm -rf /var/lib/apt/lists/*
+
+WORKDIR /src/python
+RUN wget https://www.python.org/ftp/python/3.8.5/Python-3.8.5.tgz -O python-src.tar.gz && \
+    tar xzvf python-src.tar.gz --strip-components=1 && \
+    rm python-src.tar.gz && \
+    ./configure --enable-optimizations --prefix=/opt/python-3.8.5 && \
+    make && \
+    make install && \
+    rm -rf /src/python
+WORKDIR / 
+ENV PATH="/opt/python-3.8.5/bin:${PATH}"
+
+RUN python3 -m pip install pip==20.3
+RUN pip3 install torch==1.10.1+cu113 -f https://download.pytorch.org/whl/cu113/torch_stable.html
```

Comment:
^ This will probably not work on older GPUs. Perhaps we should state it in the documentation somewhere?

Reply:
I'm thinking of using build args for this. However, I don't know the exact configuration for each GPU. I used an RTX3080, I think that we should just expose the choice of Torch versions as build arg, and then let people choose themselves. We can set a default version that works for most GPU's.

But I don't think GPU with less than 12GB VRAM works anyway, so it narrows down the old GPU's supported, depending on how old we're talking about!

Reply:
I wonder, is there a better way to get the right Torch version supporting the GPU of the machine running the image builder I'm not aware of? Currently, all my Dockerfiles are configured for my GPU, but it makes it less portable.